Mathlib/RingTheory/Bialgebra/Basic.lean

```diff
@@ -156,3 +156,32 @@ instance toBialgebra : Bialgebra R R where
   comul_one := rfl
 
 end CommSemiring
+
+namespace Bialgebra
+
+variable {R A : Type*} [CommSemiring R] [Semiring A] [Algebra R A]
+
+/-- If `R` is a commutative semiring and `A` is an `R`-algebra,
+then `Bialgebra.ofAlgHom` consumes the counit and comultiplication
+as algebra homomorphisms that satisfy the coalgebra axioms to define
+a bialgebra structure on `A`. -/
+noncomputable
+abbrev ofAlgHom (comul : A →ₐ[R] (A ⊗[R] A)) (counit : A →ₐ[R] R)
```

Comment:
Can you mention this in the module docstring, since `mk'` is mentioned there too?